### ISSUE TITLE
how to impose lower and upper bound on a cma strategy? 

### ISSUE THREAD

**Issue by kaurao** [DEAP/deap#500]
I would like to impose upper and lower bounds on a float list representation and use the cma strategy to find a solution. I search the docs and googled around but so far has not found a solution  apart from a suggestion of using the `cma` package instead.

**Comment by fmder** [comment]
You are right the current implementation of CMA-ES does not handle boundaries. However, in the devel branch there is an Adaptive version with constraints. https://github.com/DEAP/deap/blob/e8654d832458351d7811fb20bec81c7d1d5cdbce/deap/cma.py#L552
